src/main/java/com/github/kaktushose/jda/commands/entities/JDACommandsBuilder.java

```diff
@@ -35,6 +36,10 @@
     private ArgumentParser argumentParser;
     private EmbedFactory embedFactory;
     private HelpMessageSender helpMessageSender;
+    private boolean isRedisEnabled;
+    private String redisHost;
+    private Integer redisPort;
```

Comment:
usage of int wrapper class intended?

Reply:
Usage was intended due to the possibility for the user not to use redis. I did not want to set the port to something random. As of now I am working on specifying redis options in the settings.yml, therefore it won't be needed in the future anymore.